docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Build a .NET for Apache Spark application on Windows
+description: Learn how to build your .NET for Apache Spark application on Windows.
+ms.date: 01/02/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+
+# Learn how to build your .NET for Apache Spark application on Windows
+
+## Prerequisites
+
+If you already have all the prerequisites, skip to the [build](#build) steps below.
+
+  1. Download and install the **[.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** - installing the SDK will add the `dotnet` toolchain to your path. .NET Core 2.1, 2.2 and 3.1 are supported.
+  2. Install **[Visual Studio 2019](https://www.visualstudio.com/downloads/)** (Version 16.3 or later). The Community version is completely free. When configuring your installation, include these components at minimum:
+     * .NET desktop development
+       * All Required Components
+         * .NET Framework 4.6.1 Development Tools
+     * .NET Core cross-platform development
+       * All Required Components
+  3. Install **[Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)**. 
+     - Select the appropriate version for your operating system e.g., jdk-8u201-windows-x64.exe for Win x64 machine.
+     - Install using the installer and verify you are able to run `java` from your command-line.
+  4. Install **[Apache Maven 3.6.0+](https://maven.apache.org/download.cgi)**.
+     - Download [Apache Maven 3.6.0](http://mirror.metrocast.net/apache/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.zip).
+     - Extract to a local directory e.g., `c:\bin\apache-maven-3.6.0\`.
+     - Add Apache Maven to your [PATH environment variable](https://www.java.com/en/download/help/path.xml) e.g., `c:\bin\apache-maven-3.6.0\bin`.
+     - Verify you are able to run `mvn` from your command-line.
+  5. Install **[Apache Spark 2.3+](https://spark.apache.org/downloads.html)**.
```

Comment:
Same comment as Ubuntu how-to: do they have to use 2.3? Could we list other compatible Spark versions?